doc/rst/sw_midi.rst

```diff
@@ -11,7 +11,7 @@ The MIDI core implements a 31250 baud UART (8-N-1) for both input and output. It
     - UART Rx peripheral.
     - Packing of received MIDI bytes into USB MIDI messages/events.
 
-It is connected via a channel to the Endpoint Buffer core meaning that it can be placed on any XCORE tile in the system subject to resource availability. The channel uses an optimised low level protocol meaning that it always occupies a switch path.
+It is connected via a channel to the Endpoint Buffer core meaning that it can be placed on any XCORE tile in the system subject to resource availability.
```

Comment:
```suggestion
It is connected via a channel to the Endpoint Buffer core meaning that it can be placed on any `xcore` tile in the system subject to resource availability.
```

Reply:
done